admin_manual/maintenance/manual_upgrade.rst

```diff
@@ -2,83 +2,122 @@
+Copy the newly unpacked ownCloud server directory and its contents, to the original location of your existing server, so that your installation remains in the existing location. 
+::
 
-5. Unpacking the new archive creates a new ``owncloud`` directory populated 
-   with your new server files. Copy this directory and its contents to the 
-   original location of your old server, for example ``/var/www/``, so that 
-   once again you have ``/var/www/owncloud``.
+  # Assumes that the new release was unpacked in /tmp/
+  mv /tmp/owncloud /var/www/
 
-6. Copy the ``config.php`` file from your old ownCloud directory to your new 
-   ownCloud directory.
+Setup the New Installation
+--------------------------
 
-7. If you keep your ``data/`` directory in your ``owncloud/`` directory, copy 
-   it from your old version of ownCloud to your new ``owncloud/``. If you keep 
-   it outside of ``owncloud/`` then you don't have to do anything with it, 
-   because its location is configured in your original ``config.php``, and 
-   none of the upgrade steps touch it.
+With the new source files in place of the old ones, next copy the ``config.php`` file from your old ownCloud directory to your new ownCloud directory.
+::
 
-8. If you are using 3rd party applications, look in your new ``owncloud/apps/`` 
-   directory to see if they are there. If not, copy them from your old ``apps/``
-   directory to your new one. Make sure the directory permissions of your third
-   party application directories are the same as for the other ones.
+  cp -v /var/www/owncloud-old/config/config.php /var/www/owncloud/config/config.php
 
-9. Restart your Web server.
+If you keep your ``data/`` directory *inside* your ``owncloud/`` directory, copy it from your old version of ownCloud to your new version. 
+If you keep it *outside* of your ``owncloud/`` directory, then you don't have to do anything with it, because its location is configured in your original ``config.php``, and none of the upgrade steps touch it.
 
-10. Now launch the upgrade from the command  line using ``occ``, like this 
-    example on CentOS Linux::
-    
-     sudo -u apache php occ upgrade
+If you are using 3rd party applications, look in your new ``/var/www/owncloud/apps/`` directory to see if they are there. 
+If not, copy them from your old ``apps/`` directory to your new one, and make sure that the directory permissions are the same as for the other ones.
 
-  The optional parameter to skip migration tests during this step was removed in oC 9.2.
-  See :ref:`migration_test_label` for background information.
+With all that done, restart your web server.
+::
 
-  See :doc:`../configuration/server/occ_command` to learn more.
+  sudo service apache2 start
+
+Upgrade the Installation
+------------------------
+
+With the new setup in place, next launch the upgrade process from the command line.
+::
+    
+  # Here is an example on CentOS Linux
+  sudo -u apache php occ upgrade
```

Comment:
on line 109 you use the debian webserver user: sudo -u www-data php occ maintenance:mode --off
I would recommend using here the same to be identical, else this is imho confusing

Reply:
Well spotted. Thanks.